lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -719,7 +807,8 @@ export class StandardController implements IController {
                             e instanceof NativeAuthError &&
                             isFatalNativeAuthError(e)
                         ) {
-                            this.nativeExtensionProvider = undefined; // If extension gets uninstalled during session prevent future requests from continuing to attempt
+                            this.platformAuthProvider = undefined; // If extension gets uninstalled during session prevent future requests from continuing to attempt
+                            this.platformAuthType = Constants.EMPTY_STRING;
```

Comment:
Please don't use `Constants.EMPTY_STRING` it is far more expensive from a size POV.
```suggestion
                            this.platformAuthType = "";
```

Reply:
This variable has been removed from this class